src/App.vue

```diff
@@ -1,3 +1,9 @@
 <template>
   <router-view></router-view>
 </template>
+
+<style>
+body {
+  background-color: #faf7f7 !important;
```

Comment:
app.vue doesn't contain _body_ tag inside so there is no sense to add the styles here